src/DataLayer/dataSources/DataSource.ts

```diff
@@ -1,25 +1,95 @@
 // (C) 2007-2018 GoodData Corporation
-import { AFM } from '@gooddata/typings';
+import { AFM, Execution } from '@gooddata/typings';
 import * as stringify from 'json-stable-stringify';
 import { IDataSource } from '../interfaces/DataSource';
 
 export {
     IDataSource
 };
 
+export const SERVER_SIDE_COLUMN_LIMIT = 1000;
+export const SERVER_SIDE_ROW_LIMIT = 1000;
```

Comment:
exporting `SERVER_SIDE_ROW_LIMIT, SERVER_SIDE_COLUMN_LIMIT` for testing does not sound like safe way to test the limitations :) as change does not break any test

please make it private and use separate constant in tests